scib/integration.py

```diff
@@ -297,14 +295,14 @@ def runScanvi(adata, batch, labels):
     return adata
 
 
-def runMNN(adata, batch, hvg=None):
+def mnnpy(adata, batch, hvg=None):
```

Comment:
Must call this mnn I think... mnnpy is the name of the module, but not the integration method.